src/CMakeLists.txt

```diff
@@ -72,13 +73,31 @@ set(BOT_SOURCES
     WorkerManager.cpp
     WorkerManager.h
     main.cpp
+    ${PROJECT_SOURCE_DIR}/CMakeLists.txt
+    ${PROJECT_SOURCE_DIR}/README.md
```

Comment:
Well, usually you shouldn't do this as CMakeLists.txt and README.md are not source code files. As a result of this change the specified files will be put to input of compiler.

This problem should be solved in another way to avoid possible errors/warnings from compilers.